index.d.ts

```diff
@@ -1,4 +1,6 @@
 declare module "fhirpath" {
+  export function parse(path: string);
```

Comment:
Let's remove definition of parse from types annotation, because it is not documented in the readme, and it seems that it is internally used function.